src/app/visual/flyout/flyout-visual.component.ts

```diff
@@ -39,4 +41,25 @@ export class FlyoutVisualComponent {
       }]
     });
   }
+
+  public openFlyoutWithRowIterators(record: any, previousIsDisabled: boolean, nextIsDisabled: boolean) {
+    this.flyout = this.flyoutService.open(FlyoutDemoComponent, {
+      providers: [{
+        provide: FlyoutDemoContext,
+        useValue: record
+      }],
+      rowIterator: {
+        previousIsDisabled: previousIsDisabled,
```

Comment:
`previousButtonDisabled`

It might be easier for people to remember the property name if we use the pattern set by the HTML standard for the `disabled` attribute.

Reply:
this is done